### ISSUE TITLE
FR: Is there a preference to change book sort order to Title?

### ISSUE THREAD

**Issue by coop555** [koreader/koreader#9522]
Thank you for an excellent reader.

When opening a directory of books in koreader, for the first 1 second, the books are listed by the book's Filename and are sorted as such.  

One second later, the books change to listed by Title instead of Filename.  However the sort order remains by the hidden Filename not the Title now shown on the screen.   

I use Calibre's metadata plugboard save_to_disk template to change the books' metadata Titles to enable Title sorting in the manner I wish (ie  "Harry Potter [1] 1997 Harry Potter and the Philosopher's Stone".  This enables nice sorting in the Kindle's standard book list viewer.

Is there a preference in koreader to sort by Title to make the sort match what is displayed?

**Comment by NiLuJe** [comment]
Nope, the file-manager is primarily file-based, metadata is just used for display purposes.

(So, yeah, in a calibre, workflow, the save_to-disk template is the preferred way to handle this).

**Comment by NiLuJe** [comment]
It won't be the FM, but since this is a calibre workflow, you can use the Calibre metadata search to search/Brows by tags/metadata.

**Comment by coop555** [comment]
Thanks.  I had missed the File Cabinet had a menu when browsing books.  I didn't realize koreader had the option to display book Filenames instead of Titles.   That enables a good workaround for me.  

Instead of using Calibre > plugboards > save_to_disk, I used Calibre > Saving Books to Disk feature.  This way Calibre modifies the filename instead of the Title.

Then I set koreader > File Cabinet > "Detailed list with cover images and filenames".    I picked this one as it shows a much longer filename than the other options. 

Is there any way to make that File Cabinet menu be what appears when I click the File Cabinet when reading a book?  Because there are tremendous number of options in the menus, I find myself constantly clicking them to familiarize myself with them.  And each time I accidentally click the File Cabinet, it is jarring to me to have found I left the book and have all the menus be gone/ change and have to re-open the book.  Ideally I'd have that same menu there with one additional top option added to it:  A entry for "Home" which takes me back to my books.

**Comment by NiLuJe** [comment]
The reader and the FM are technically two separate applications, hence the differing menus ;).

**Comment by coop555** [comment]
Understood, thank you.  Last question and sorry to steer it unrelated topics, but is there any option to remove the FM icon while in reader mode (or change what it does), and instead switch to the Home directory of the FM via a custom shortcut?

**Comment by NiLuJe** [comment]
Patching out the icon is quite likely doable with a user patch, and I imagine "Go Home" is in Dispatcher, so it should be available in the Gesture manager (I have checked neither of those facts ;p).

**Comment by coop555** [comment]
thank you
